addons/common/jobs/api/src/test/java/org/kie/kogito/jobs/api/event/AbstractJobCloudEventTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.kogito.jobs.api.event;
+
+import java.net.URI;
+import java.time.ZonedDateTime;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+abstract class AbstractJobCloudEventTest<E extends JobCloudEvent<T>, T> {
```

Comment:
I don't see the second type parameter used anywhere, do we need it? Wouldn't something like:
```suggestion
abstract class AbstractJobCloudEventTest<E extends JobCloudEvent<?>> {
```
also work?

Reply:
added, thanks.